pandas/io/formats/_color_data.py

```diff
@@ -0,0 +1,151 @@
+# source: matplotlib._color_data (3.3.3)
```

Comment:
can you add some more commentary here, why we need this and we don't want to depend on using mpl for this data.

Reply:
done

Reply:
thanks. Do we have any need for this in plotting itself? @ivanovmg @charlesdong1991 @jorisvandenbossche